buildMREUnityRuntimeDLL.bat

```diff
@@ -3,20 +3,20 @@
 
 for /f "usebackq tokens=*" %%i in (`"%~dp0tools\vswhere" -products * -requires Microsoft.Component.MSBuild -property installationPath`)	 do (
   set VSInstallDir=%%i
-  if exist "%%i\MSBuild\15.0\Bin\MSBuild.exe" goto FoundVisualStudio
+  if exist "%%i\MSBuild\Current\Bin\MSBuild.exe" goto FoundVisualStudio
 )
-echo MSBuild not found - please install Visual Studio 2017
+echo MSBuild not found - please install Visual Studio 2019
```

Comment:
I'd prefer this be changed to `2017 or later`, because Unity itself is officially still on 2017. Same comment applies below.

Reply:
Got it. thank you.